tests/ocf-url_parse-path-absolute/EPUB/package.opf

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-03-05</dc:date>
+    <dc:description>Parse a path-absolute URL, i.e., starting with a slash, from the content to an image. The folder hierarchy containing the image starts at the root level.</dc:description>
```

Comment:
"beginning with a leading slash" for consistency with https://github.com/w3c/epub-tests/blob/main/tests/ocf-url_link-path-absolute/EPUB/package.opf

Reply:
yep